code/messaging/src/main/java/com/adobe/marketing/mobile/aepcomposeui/aepui/components/AepTextComposable.kt

```diff
@@ -11,76 +11,74 @@
 
 package com.adobe.marketing.mobile.aepcomposeui.aepui.components
 
-import androidx.compose.foundation.layout.fillMaxWidth
-import androidx.compose.material.Text
+import androidx.compose.material3.Text
```

Comment:
I think  we can move the subpackages of  `aepui` under `aepcomposeui` and remove `aepui`.
- `com.adobe.marketing.mobile.aepcomposeui.components`
- `com.adobe.marketing.mobile.aepcomposeui.style`
- `com.adobe.marketing.mobile.aepcomposeui.util`
- `com.adobe.marketing.mobile.aepuitemplates.uimodels`

Reply:
Done

Reply:
Also moved `uimodels` inside `aepcomposeui` as discussed in the previous PR discussion
https://github.com/adobe/aepsdk-messaging-android/pull/281#pullrequestreview-2375729923
We can rename the packages and classes more succinctly now if needed